src/components/Channel/ChannelSide.jsx

```diff
@@ -38,7 +44,7 @@ export default function ChannelSide() {
         ))}
       </AudienceWrapper>
       <ButtonWrapper>
-        {currentUser?.currentUser._id === host._id ? (
+        {currentUser?.currentUser === host._id ? (
```

Comment:
```suggestion
        {currentUser?._id === host._id ? (
```
optional chaning의 용법을 잘못 사용하고계신 것 같아요!

[문서](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)를 참고해보세용~